Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -17,29 +23,65 @@ namespace Content.Server.GameObjects.Components.Damage
     public class DestructibleComponent : RuinableComponent, IDestroyAct
     {
         [Dependency] private readonly IEntitySystemManager _entitySystemManager = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly IRobustRandom _random = default!;
 
         protected ActSystem ActSystem;
 
         /// <inheritdoc />
         public override string Name => "Destructible";
 
         /// <summary>
-        ///     Entity spawned upon destruction.
+        /// Entities spawned on destruction plus the min and max amount spawned.
         /// </summary>
-        public string SpawnOnDestroy { get; private set; }
+        public Dictionary<string, List<int>> SpawnOnDestroy { get; private set; }
```

Comment:
Maybe instead of a list of int, use a tuple/struct? That way you can't have more than two values.

Reply:
Tuples would be perfect here C# wise, but on the YAML side they're shit. Structs seem like they would be too clumsy code-wise and possibly just as shit as tuples YAML-wise (I didn't actually get that far because I didn't like how it would look on the C# side of things).

A list is probably the least worst option here. It's not too annoying to define in YAML, though it could be better, and if some doofus puts extra entries in there I don't think it's a big deal because they're just unused. Unless there's some way to make non-randomized drops in YAML only require a single scalar I don't think there's a better option.